tasks/blacklight.rake

```diff
@@ -11,10 +11,15 @@ require 'rubocop/rake_task'
 RuboCop::RakeTask.new(:rubocop)
 
 desc "Run test suite"
-task ci: ['blacklight:generate'] do
+task :ci do
+  system "docker-compose up -d solr"
+
+  Rake::Task['engine_cart:generate'].invoke
+
   within_test_app do
-    system "RAILS_ENV=test rake blacklight:index:seed"
+    system "RAILS_ENV=test bin/rake blacklight:index:seed"
```

Comment:
Use generated binstubs. See https://github.com/rails/rails/blob/v6.0.2.1/railties/lib/rails/app_loader.rb#L13-L14

Not necessary for this work, so I'm happy to undo this if folks would prefer.